modules/installation-osp-local-disk-deployment.adoc

```diff
@@ -0,0 +1,365 @@
+// Module included in the following assemblies:
+//
+// * installing_openstack/deploying-openstack-with-rootVolume-etcd-on-local-disk.adoc
+
+:_mod-docs-content-type: Procedure
+[id="installation-osp-local-disk-deployment_{context}"]
+= Deploying {rh-openstack} on local disk
+
+.Prerequisites
+
+* You have an OpenStack cloud with a working Cinder, and it has at least 75 GB of available storage to accommodate 3 root volumes for the OpenShift control plane.
+
+* The OpenStack cloud must be deployed with Nova ephemeral storage that uses a local storage backend and not RBD.
+
+
+.Procedure
+
+[WARNING]
+====
+Failure to move to etcd immidiately after deploying a control plane and before any workload is running can result in the loss of the control plane node.
```

Comment:
not just the control plane, but can also bring data corruption, etc. Just mention "the control plane", I think is fine.

Reply:
The issue isn't that moving etcd is dangerous, it's that this specific procedure is bad. I'd prefer something like:

This migration procedure is suitable only for testing etcd on local disk. It is not suitable for use on production clusters as it risks complete loss of the control plane in certain cases. It will be replaced with a robust migration procedure before this feature is generally available.